freud/locality.pyx

```diff
@@ -774,9 +774,15 @@ cdef class _RawPoints(NeighborQuery):
 
 
 cdef class AABBQuery(NeighborQuery):
-    R"""Use an AABB tree to find neighbors.
+    R"""Use an Axis-Aligned Bounding Box (AABB) tree to find neighbors.
```

Comment:
One thing we need to be a bit better about in freud is citations. We improved on this quite a bit when revising for 2.0, but here we are thoroughly lacking. Would you mind adding a citation of Michael Howard's paper https://www.sciencedirect.com/science/article/pii/S0010465516300182 to the bibtex file (in doc/source/reference/freud.bib) and then referencing it here? The format for inline citations that we're using now looks like :cite:`vanAnders:2014aa` (from e.g. pmft.pyx).